docs/http_api.rst

```diff
@@ -165,6 +165,66 @@ Channels
    Delete a channel.
 
 
+Logs
+^^^^
+
+.. http:get:: /channels/(channel_id:str)/logs
+
+   Get the most recent logs for a specific channel.
+
+   :query int n:
+       The amount of logs to fetch.
+
+
+   The response is a list of logs, with each log taking the following form:
+
+   :param str logger: The logger that created the log, usually the channel id.
+   :param int level:
+       The level of the logs. Corresponds to the levels found in the python
+       module :py:mod:`logging`.
+   :param float timestamp: Timestamp, in the format of seconds since the epoch.
+   :param str message: The message of the log.
+
+   In the case of an exception, the following fields may also be returned:
+
+   :param str class: The class of the exception.
```

Comment:
I wonder if we should nest these inside an `exception` sub-dict? Just thinking that `class` and `instance` by themselves don't really indicate that they are related to an exception?